tools/rapids-print-env

```diff
@@ -13,7 +13,9 @@ arch
 rapids-logger "Check python version"
 python --version
 
-rapids-logger "Check conda environment"
-conda info
-conda config --show-sources
-conda list --show-channel-urls
+if conda &> /dev/null; then
```

Comment:
I think we want to use something like this?
```suggestion
if command -v conda &> /dev/null; then
```

Reply:
Is there any advantage to using `command -v` vs. just trying the `conda` command?

Reply:
We should not actually invoke `conda`.

`command -v` is cheaper (basically zero time vs. 0.5 seconds to invoke `conda`).

Reply:
I see. Okay. Good point. I will test the change locally real quick

Reply:
updated in [988969c](https://github.com/rapidsai/gha-tools/pull/109/commits/988969cfa20643bfbfba6ed19c15c68938c63eb8)